index.html

```diff
@@ -38,6 +39,156 @@
+      </div>
+      <div class="diffrent-work">
+      <div class="worktwo">
+        <p class="work-two-title">Profesional Art Printing Data</p>
+        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
+          or sign-ups required. has been the industry's standard</p>
+          <div class="see-two-box"><p class="see-two">see Project</p></div>
+          <ul class="languages-work">
+            <li class="html-box"><p class="code-name">html</li>
+            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+            <li class="ruby-box"><p class="code-name">Ruby</p></li>
+          </ul>
+          
+       </div>
+          
+       <div class="worktwo">
+        <p class="work-two-title">Profesional Art Printing Data</p>
+        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
+          or sign-ups required. has been the industry's standard</p>
+          <div class="see-two-box"><p class="see-two">see Project</p></div>
+          <ul class="languages-work">
+            <li class="html-box"><p class="code-name">html</p></li>
+            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+            <li class="ruby-box"><p class="code-name">Ruby</p></li>
+          </ul>
+          
+       </div>
+       <div class="worktwo">
+        <p class="work-two-title">Profesional Art Printing Data</p>
+        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
+          or sign-ups required. has been the industry's standard</p>
+          <div class="see-two-box"><p class="see-two">see Project</p></div>
+          <ul class="languages-work">
+            <li class="html-box"><p class="code-name">html</p></li>
+            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+            <li class="ruby-box"><p class="code-name">Ruby</p></li>
+          </ul>
+          
+       </div>
+       <div class="worktwo">
+        <p class="work-two-title">Profesional Art Printing Data</p>
+        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
+          or sign-ups required. has been the industry's standard</p>
+          <div class="see-two-box"><p class="see-two">see Project</p></div>
+          <ul class="languages-work">
+            <li class="html-box"><p class="code-name">html</p></li>
+            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+            <li class="ruby-box"><p class="code-name">Ruby</p></li>
+          </ul>
+          
+       </div>
+       <div class="worktwo">
+        <p class="work-two-title">Profesional Art Printing Data</p>
+        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
+          or sign-ups required. has been the industry's standard</p>
+          <div class="see-two-box"><p class="see-two">see Project</p></div>
```

Comment:
- [ ] Kindly ensure that all your "See Project" buttons fit exactly the projects card image. From your design, I can see that the buttons are not properly aligned with the project card images. Kindly fix it 👍🏽


| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/192101249-a741d799-506a-41ca-8c85-9c9bdb321d57.png)  | ![image](https://user-images.githubusercontent.com/63473584/192101267-5d8eb6b3-1a58-47f1-84cd-033507f587d3.png)  |

Reply:
I've been having real difficulties with that any suggestion on how to fix the issue

Reply:
A way you can fix it is to specify a position: relative to the project card container, then the  button can take absolute positioning with button :0; left: 0; so that the button will be properly aligned. 

You can also use flexbox